15 error Error: Command failed: git remote update
15 error fatal: unable to access 'https://github.com/ht1131589588/rayslog.git/': Could not resolve host: github.com
15 error error: Could not fetch origin
15 error
15 error Fetching origin
15 error
15 error     at makeError (D:\hutao\node\nodejs\node_global\node_modules\lerna\node_modules\_execa@1.0.0@execa\index.js:174:9)
15 error     at Function.module.exports.sync (D:\hutao\node\nodejs\node_global\node_modules\lerna\node_modules\_execa@1.0.0@execa\index.js:338:15)
15 error     at Object.execSync (D:\hutao\node\nodejs\node_global\node_modules\lerna\node_modules\_@lerna_child-process@3.13.0@@lerna\child-process\index.js:22:16)
15 error     at updateRemote (D:\hutao\node\nodejs\node_global\node_modules\lerna\node_modules\_@lerna_version@3.13.1@@lerna\version\lib\is-behind-upstream.js:26:16)
15 error     at isBehindUpstream (D:\hutao\node\nodejs\node_global\node_modules\lerna\node_modules\_@lerna_version@3.13.1@@lerna\version\lib\is-behind-upstream.js:11:3)
15 error     at VersionCommand.initialize (D:\hutao\node\nodejs\node_global\node_modules\lerna\node_modules\_@lerna_version@3.13.1@@lerna\version\index.js:149:9)
